Assets/Plugins/iOS/Shopify/Checkout/Checkout.WebContentView.swift

```diff
@@ -0,0 +1,142 @@
+//
+//  Checkout.WebContentView.swift
+//  UnityBuySDK
+//
+//  Created by Shopify.
+//  Copyright © 2017 Shopify Inc. All rights reserved.
+//
+//  Permission is hereby granted, free of charge, to any person obtaining a copy
+//  of this software and associated documentation files (the "Software"), to deal
+//  in the Software without restriction, including without limitation the rights
+//  to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+//  copies of the Software, and to permit persons to whom the Software is
+//  furnished to do so, subject to the following conditions:
+//
+//  The above copyright notice and this permission notice shall be included in
+//  all copies or substantial portions of the Software.
+//
+//  THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+//  IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+//  FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+//  AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+//  LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+//  OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+//  THE SOFTWARE.
+//
+
+import UIKit
+import WebKit
+
+extension Checkout {
+
+// View contents of the checkout view controller.
+class WebContentView: UIView {
+    var isFinished = false {
+        didSet {
+            actionButton.checkoutState = isFinished ? .finished : .checkingOut
+        }
+    }
+    
+    private let cornerRadius: CGFloat = 3
+    private let viewMargin: CGFloat = 10
+    private let buttonHeight: CGFloat = 46
+    private let loadingIndicatorOffset: CGFloat = 5
+    private let loadingIndicatorTiming: TimeInterval = 0.3
+    
+    private(set) var containerView: UIView
+    private(set) var webView: WKWebView
+    private(set) var actionButton: Button
+    private(set) var progressIndicator: WebPageProgressIndicator
```

Comment:
Do these all have to be `var`? Are they getting modified anywhere? Can they potentially be converted to constant?